Makefile

```diff
@@ -1,117 +1,58 @@
-# useful paths
-SRC_DIR = src
-HDR = $(SRC_DIR)/cs50.h
-SRC = $(SRC_DIR)/cs50.c
+# parse version from debian/changelog
+VERSION := $(shell dpkg-parsechangelog --show-field Version | sed s/-0ubuntu1//)
```

Comment:
Removed and ignored `changelog`, moved version to `Makefile`, and `echo`ing changelog entry when `make deb` per a17651d!